letsdrill.rb

```diff
@@ -1,13 +1,29 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
+ grade = ""
+  grade_ranges = {
+  	"A" => 101,
+  	"B" => 90,
+  	"C" => 80,
+  	"D" => 70,
+  	"F" => 60,
+  }
 
+  grade_ranges.each do |letter, upper_bound|
+  	if (integer < upper_bound)
+  		grade = letter
+  	end
+  end
+  return grade
```

Comment:
Interesting approach.  If you started from the bottom and worked your way up (rather than starting from the top), you might be able to avoid that awkward `101`.